nix/sources.nix

```diff
@@ -0,0 +1,138 @@
+# This file has been generated by Niv.
```

Comment:
I've never had to maintain this file.  I suppose if there are ever breaking changes, delete it and `niv init`.  Or keep `niv` pinned. :smile: